docs/develop/development_team.rst

```diff
@@ -99,13 +102,12 @@ additional surface observational datasets, adding the capability of pairing
 to aircraft observations, and expanding plotting and statistics for analysis 
 of aircraft data.
 
-**Jun Zhang**
-I am a postdoc at ACOM NCAR. My research interests are at the UTLS region.
+**Jun Zhang:**
+I am a postdoc at NSF NCAR/ACOM. My research interests are at the UTLS region.
 I am hoping to add cross-section plots (lat vs height) and also to incorporate MLS data
 for comparison.
 
-
-
-
-
-
+**Pablo Lichtig:**
+I am a postdoc at NSF NCAR/ACOM. My development plans include adding TEMPO support,
+working with remote sensing instruments from the surface and adding support for BoulderAir
+and other surface networks.
```

Comment:
@blychs  Please find my info for for development_team.rst:

Quazi Ziaur Rasool

I am a research scientist at CIRES/NOAA CSL helping with MELODIES-MONET development. My development plans include incorporating new comparisons (already tested)  for aircraft observation datasets such as FIREX, ASIA-AQ
AEROMMA with model output from WRF-Chem (regional), UFS-AQM (global); and expanding the analysis of model-aircraft comparisons with new visualizations (for e.g., curtain plots, spatial plots along aircraft tracks, vertical profiles, time series with altitude) and new observations or model data pairing of interest.